applications/metadata.schema.json

```diff
@@ -54,9 +54,115 @@
         "ranking",
         "dependencies"
       ]
+    },
+    "operator": {
```

Comment:
Should we reference existing operator/extension schema rather than redefining here?

https://github.com/nvidia-holoscan/holohub/blob/main/operators/metadata.schema.json
https://github.com/nvidia-holoscan/holohub/blob/main/gxf_extensions/metadata.schema.json

Reply:
Not sure how to do that...

Reply:
Should be able to use a `$ref` field similar to how those schema refer to utility definitions for common fields (name, etc).

Something like:
```
"$ref": "holohub/operator/v1#/properties/operator"
```

Probably not worth spending much time on though, if copy/paste lets us move forward then we can stick with that.